template/README.md

```diff
@@ -504,6 +506,32 @@ If the `proxy` option is **not** flexible enough for you, alternatively you can:
 * Enable CORS on your server ([here’s how to do it for Express](http://enable-cors.org/server_expressjs.html)).
 * Use [environment variables](#adding-custom-environment-variables) to inject the right server host and port into your app.
 
+## Using Promises
+
+This project includes a [Promise polyfill](https://github.com/then/promise) which provides a full implementation of Promises/A+.
+
+A Promise represents the eventual result of an asynchronous operation, you can find more information about Promises [here](https://www.promisejs.org/) and [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise).
```

Comment:
Let’s make this go right after previous sentence without a paragraph break in between.